client/landing/stepper/declarative-flow/new-hosted-site-flow.ts

```diff
@@ -26,7 +26,17 @@ const hosting: Flow = {
 	name: NEW_HOSTED_SITE_FLOW,
 	isSignupFlow: true,
 	useSteps() {
+		const query = useQuery();
+		const isPatnerBundle = query.has( 'partnerBundle' );
 		return [
+			...( isPatnerBundle
```

Comment:
The idea of injecting dynamically the domain step seems cool! I wonder if we should make it abstract and make it use a showDomainStep query parameter instead.

Reply:
That would be more flexible, as it would allow some partners to show the domain step while others might not show it. I will go ahead and implement that instead

Reply:
I have added a small reusable hook here https://github.com/Automattic/wp-calypso/pull/95721/commits/4b7b1ce42df5ddb74079965b8a63654e1315dde0#diff-cc59aa46143159b6a6ca515435b7e71073c365342162deb1a2de7f43f1dee0fdR25 to be reused in the `goBack` functionality below